x-pack/plugins/ml/public/application/explorer/anomaly_timeline.tsx

```diff
@@ -35,6 +35,54 @@ import { ExplorerNoInfluencersFound } from './components/explorer_no_influencers
 import { SwimlaneContainer } from './swimlane_container';
 import { AppStateSelectedCells, OverallSwimlaneData, ViewBySwimLaneData } from './explorer_utils';
 import { NoOverallData } from './components/no_overall_data';
+import { HelpPopover, HelpPopoverButton } from '../components/help_popover/help_popover';
+
+const AnomalyTimelineHelpPopover = () => {
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+
+  return (
+    <HelpPopover
+      anchorPosition="upCenter"
+      button={
+        <HelpPopoverButton
+          onClick={() => {
+            setIsPopoverOpen(!isPopoverOpen);
+          }}
+        />
+      }
+      closePopover={() => setIsPopoverOpen(false)}
+      isOpen={isPopoverOpen}
+      title={i18n.translate('xpack.ml.explorer.anomalyTimelinePopoverTitle', {
+        defaultMessage: 'Anomaly timelines',
+      })}
+    >
+      <p>
+        {i18n.translate('xpack.ml.explorer.anomalyTimelinePopoverBasicExplanation', {
+          defaultMessage:
+            'Swim lanes provide an overview of the buckets of data that have been analyzed within the selected time period. You can view an overall swim lane or view them by job or influencer.',
+        })}
```

Comment:
According to kibana guidelines `FormattedMessage` component is preferred for handling translation strings in React components:
```suggestion
        <FormattedMessage
          id="xpack.ml.explorer.anomalyTimelinePopoverBasicExplanation"
          defaultMessage="Swim lanes provide an overview of the buckets of data that have been analyzed within the selected time period. You can view an overall swim lane or view them by job or influencer."
         />
```

Reply:
Thanks!  I've made that change in the rest of the strings too.